classes/Misc.php

```diff
@@ -494,7 +494,7 @@ function getDatabaseAccessor($database, $server_id = null) {
 				exit;
 			}
 			$this->setServerInfo('platform', $platform, $server_id);
-			$this->setServerInfo('pgVersion', $_connection->conn->pgVersion, $server_id);
+			$this->setServerInfo('pgVersion',/* $_connection->conn->pgVersion*/ "unknown version", $server_id);
```

Comment:
This variable wasn't defined and I'm not sure where it ever lived. I simply removed it and phppgadmin seems to still properly report the version.

Reply:
I believe #125 is a better fix for that one.

Reply:
Agreed